cell_classification/segmentation_data_prep_test.py

```diff
@@ -334,5 +367,115 @@ def test_get_marker_activity_mask():
         ).all()
 
 
+def test_tile_example():
+    example = {
+        "mplex_img": np.random.rand(512, 512, 3).astype(np.float32),
+        "binary_mask": np.random.randint(0, 2, [512, 512, 1]).astype(np.uint8),
+        "instance_mask": np.random.randint(0, 10000, [512, 512]).astype(np.uint16),
+        "marker_activity_mask": np.random.randint(0, 2, [512, 512, 21]).astype(np.uint8),
+        "dataset": "test_dataset",
+        "platform": "mibi",
+        "cell_types": ["CD11c", "CD11b", "CD11a"],
+        "marker": "CD11c",
+    }
+    data_prep = prep_object(tile_size=[128, 128], stride=[128, 128])
+    tiled_examples = data_prep.tile_example(example)
+
+    # check if the correct number of tiles got returned
+    assert len(tiled_examples) == 16
+
+    # check if the correct spatial dimensions got returned and dtype is correct
```

Comment:
Can you also add a check that the first tiled example is equal to the top left corner of the original untiled image, last one is bottom right, or something along those lines?

Reply:
Yep, I added tests for both cases for all spatial keys.